libs/ui-watt/src/lib/components/input/timepicker/watt-timepicker.component.ts

```diff
@@ -43,6 +52,28 @@ import { WattRangeInputService } from '../shared/watt-range-input.service';
 const hoursMinutesFormat = 'HH:MM';
 const hoursMinutesPlaceholder = 'HH:MM';
 
+// Show slider initially as "00:00 - 23:59"
+const initialSliderValue: WattSliderValue = { min: 0, max: 1439 };
```

Comment:
suggestion: consider moving the number `1439` to a variable, e.g. `_23_59_asNumberOfMinutes` or similar, to give it some meaning.

Reply:
f9f17fe739bf6bb0d36e3fbe0c6fa6dd33369aae